src/components/Calculator.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import '../App.css';
+import Result from './Result';
+
+function Calculator() {
+  return (
+    <div className="calculator">
```

Comment:
- [ ] I would implore you to add at least one child component to Calculator.js just as it is required in the project requirements. With this, you will master the use of components in React and similar libraries.
![image](https://user-images.githubusercontent.com/86200356/221872910-97fbcd9c-3921-473f-af88-b6e0e5d3bd78.png)

Reply:
resolved! ✔️